multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -647,17 +656,18 @@ def _hyperparameter_tune(self, hyperparameter_tune_kwargs, resources, **_fit_arg
             with open(best_k_model_path, "w") as yaml_file:
                 yaml.dump(checkpoints_paths_and_scores, yaml_file, default_flow_style=False)
 
-            last_ckpt_path = analysis.get_last_checkpoint(best_trial)
-            predictor._top_k_average(
-                model=predictor._model,
-                save_path=best_trial_path,
-                last_ckpt_path=last_ckpt_path,
-                minmax_mode=mode,
-                is_distill=is_distill,
-                top_k_average_method=predictor._config.optimization.top_k_average_method,
-                val_df=_fit_args["val_df"],
-                validation_metric_name=predictor._validation_metric_name,
-            )
+            last_ckpt_path = analysis.get_last_checkpoint(best_trial)._local_path
```

Comment:
`last_ckpt_path ` seems unused. Is it the same as the `last_ckpt_dir`?

Reply:
This should be removed. I was experimenting with this variable and forgot to remove it :)